packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -84,13 +92,15 @@ const createConceptDisplays = (compareConceptDisplays) => {
   return null;
 };
 
+let idForDatetimeDisplays;
 const createDatetimeDisplays = (compareDatetimeDisplays) => {
+  idForDatetimeDisplays = `${uuidv4()}-datetimeDisplay`;
```

Comment:
We can assign `idForDatetimeDisplays` upon creation as it is not being mutated.
```suggestion
const idForDatetimeDisplays = `${uuidv4()}-datetimeDisplay`;
const createDatetimeDisplays = (compareDatetimeDisplays) => {
```

Reply:
Updated here - https://github.com/cerner/terra-clinical/pull/868/commits/f932e12eaad49881c42630811d1619d1000807ed#diff-42afe5315e2b63b43f755f1ec567ef32b1ad903109d718db4e7526f825680f6aR94